### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by pdelvo**
I think that our readme could really use some more information. Currently the readme does not really look inviting. I added a bit more information

### CODE REVIEW

**Comment by sharwell** [comment]
> Currently the readme does not really look inviting.

I'm scared to think about what it means to have *even more* active contributors.

File: README.md [hunk 1]
```diff
@@ -6,4 +6,32 @@
...
+## Using StyleCop.Analyzers
+
+The preferable way to use the analyzers is to add the nuget package [StyleCop.Analyzers](http://www.nuget.org/packages/StyleCop.Analyzers/)
+to the project where you want to enforce StyleCop rules.
+You can also build a vsix extension from source and install it into visual studio to use it in all
+C# projects opened in visual studio.
+
+Currently the only way to configure the rules is to change the severity for them in a ruleset file.
+
+## Building
```
Comment (sharwell) [inline README.md]: This section should go in **CONTRIBUTING.md**.

File: README.md [hunk 2]
```diff
@@ -6,4 +6,32 @@
...
+
+Visual Studio 2015 RC is required for building this repository.
+The Visual Studio 2015 RC SDK is required for building the vsix extension project and for
+debugging in an experimental visual studio hive.
+
+## Contributing
```
Comment (sharwell) [inline README.md]: This section should go in **CONTRIBUTING.md**.

File: README.md [hunk 3]
```diff
@@ -6,4 +6,32 @@
 
 This repository contains an implementation of the StyleCop rules using the .NET Compiler Platform. Where possible, code fixes are also provided to simplify the process of correcting violations.
 
+## Using StyleCop.Analyzers
+
+The preferable way to use the analyzers is to add the nuget package [StyleCop.Analyzers](http://www.nuget.org/packages/StyleCop.Analyzers/)
```
Comment (sharwell) [inline README.md]: :bulb:

StyleCopAnalyzers can be installed using the NuGet Package Manager in Visual Studio 2015 RC.

image

**Comment by pdelvo** [comment]
I implemented your feedback. please have a look if this is better

**Comment by sharwell** [comment]
Looks good. I'll look over it one last time when I'm on a normal computer instead of a phone (this evening).

**Pull request merged**